src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  const clonedState = { ...state };
+  const finalState = [];
+  const deleteKey = (key) => delete clonedState[key];
+
+  for (const obj of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
obj isn't a descriptive name